libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -19,6 +19,13 @@
 #include <SRV_Channel/SRV_Channel.h>
 #include <AP_Logger/AP_Logger.h>
 
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    #define AP_MOTORS_PARAM_PREFIX "Q_M_"
+#else
+    #define AP_MOTORS_PARAM_PREFIX "MOT_"
+#endif
+
```

Comment:
This is not ideal because we have to remember to update in both places. But I don't think were going to be changing them any time soon.

Alternatively we could:

- pass down the prefix from the vehicle in the arming check function
- reverse lookup full param names from param pointers in AP Param.

Both other methods would be more complex and cost more flash.

Reply:
I think this is fine.  Simple.. easy to change.  We will probably remember and if not someone will remind us..